src/app/components/challenge/challengesubmit/challengesubmit.component.ts

```diff
@@ -260,6 +361,27 @@ export class ChallengesubmitComponent implements OnInit {
     );
   }
 
+  copyTextToClipboard(val: string) {
+    if (val === 'pip install ') {
+      val += '"evalai' + this.cliVersion + '"';
```

Comment:
Have this special cases handled at the html tag level itself. Don't do this parsing at js side.

Reply:
yeah thanks. Now set it as button value and used the template reference on the button to get the value on the TS side. Let me know if this looks good to you :)